roles-in-tech/dev-ops.md

```diff
@@ -17,75 +13,70 @@ Here are links to lessons that should be completed before this lesson.
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+*One of the motivation to join DevOps is its `Fast Career Growth`.* 
+You can become a **Release Manager**, **Project Manager**, **Automation Architect** or even a **DevOps Evangelist** by learning DevOps.
+
+<h2>Any One Can Learn DevOps</h2>
+>People from different backgrounds can learn DevOps. Even a fresher with basic knowledge of Linux and one scripting language can learn DevOps.
```

Comment:
Please give an example of a company that uses DevOps well or depends on it a lot, and how they do it.

Reply:
Okay, I am adding this on the slide as well as in my content.